Sources/XCTest/XCTestCase.swift

```diff
@@ -26,7 +26,14 @@ public typealias XCTestCaseEntry = (testCaseClass: XCTestCase.Type, allTests: [(
 
 public class XCTestCase {
 
+    /// The name of the test case, consisting of its class name and the method name it will run.
+    /// - Note: FIXME: This property should be readonly, but currently has to be publicly settable due to a
+    ///         toolchain bug on Linux. To ensure compatibility of tests between
+    ///         swift-corelibs-xctest and Apple XCTest, this property should not be modified.
```

Comment:
Oh wow, interesting... It would be great to reference a JIRA issue here. :point_up: